automation/test.sh

```diff
@@ -363,3 +364,19 @@ fi
 
 # Run functional tests
 FUNC_TEST_ARGS=$ginko_params make functest
+
+# Run REST API coverage based on k8s audit log and openapi spec
+if [ -n "$RUN_REST_COVERAGE" ]; then
+  echo "Generating REST API coverage report"
+  wget https://github.com/mfranczy/crd-rest-coverage/releases/download/v0.1.3/rest-coverage -O _out/rest-coverage
+  chmod +x _out/rest-coverage
+  AUDIT_LOG_PATH=${AUDIT_LOG_PATH-/var/log/k8s-audit/k8s-audit.log}
+  log_dest="$ARTIFACTS_PATH/cluster-audit.log"
+  cli scp "$AUDIT_LOG_PATH" - > $log_dest
+  _out/rest-coverage \
+    --swagger-path "api/openapi-spec/swagger.json" \
+    --audit-log-path $log_dest \
+    --output-path "$ARTIFACTS_PATH/rest-coverage.json" \
+    --ignore-resource-version
```

Comment:
Maybe we should somehow report the executable exit code in case it fails for some reason?
Other than that looks great to me :)
/lgtm

Reply:
In any case the exit code is reported at the end of the prow job log: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5725/pull-kubevirt-e2e-k8s-1.21-rest-coverage/1397811098802982912#1:build-log.txt%3A8343